compliance/election.go

```diff
@@ -575,6 +580,9 @@ func TestSameElectionIDFromTwoClients(c *fluent.GRIBIClient, t testing.TB, opts
 			AsResult(),
 	)
 
+	fmt.Printf("%v\n", clientA.Results(t))
+	fmt.Printf("%v\n", clientB.Results(t))
+
```

Comment:
Hi Rob, I worked with Deep on the similar issue in this testcase in the GitHub action run. Sorry about creating the race condition. I think the run failure is related to race condition in the session_param and election process. I tried to add the awaitTimeout calls after each client start. Following is the PR. Could you please check if it would help in your case?
https://github.com/deep-gajjar5/gribigo/pull/1

Reply:
Hi @mingyangcisco -- awesome, thanks for fixing this.I was debugging where the race was. I agree with you that the race condition is there -- thanks for spotting this :-) I'll close this PR.